deployment.rb

```diff
@@ -187,6 +187,18 @@ def hourofcode_hostname
     canonical_hostname('hourofcode.com')
   end
 
+  def circle_run_identifier
+    ENV['CIRCLE_BUILD_NUM'] ? "CIRCLE-BUILD-#{ENV['CIRCLE_BUILD_NUM']}-#{ENV['CIRCLE_NODE_INDEX']}" : nil
```

Comment:
Within runner.rb `S3_LOGS_PREFIX` only gets used for Cucumber logs, so I agree with @bcjordan that we're cool with all containers using the same prefix.  [In other places](https://github.com/code-dot-org/code-dot-org/blob/staging/aws/ci_build#L118) where we upload build logs they're currently being identified only by environment + timestamp; it might be prudent to add a circle prefix and/or container prefix in case we have collisions from circle contains starting up at very similar times.